tests/disposableredis/__init__.py

```diff
@@ -8,10 +8,10 @@
 import warnings
 import random
 
+REDIS_SERVER = os.environ.get('REDIS_SERVER', 'redis-server')
```

Comment:
Which value does REDIS_SERVER holds incase this env variable isn't defined?

Reply:
'redis-server', to stay compatible with the former ctor:
class DisposableRedis(object):
    def __init__(self, port=None, path='redis-server', **extra_args):